consumer/dragonboard/qualcomm-robotics-rb5/installation/board-recovery.md

```diff
@@ -1,29 +1,29 @@
 ---
-title: DragonBoard 845c Board Recovery
-permalink: /documentation/consumer/dragonboard/dragonboard845c/installation/board-recovery.md.html
+title: Qualcomm RB5 Board Recovery
+permalink: /documentation/consumer/dragonboard/qualcomm-robotics-rb5/installation/board-recovery.md.html
 ---
 
-# DragonBoard 845c Board Recovery
+# Qualcomm RB5 Board Recovery
 
-This page outlines steps needed to recover your DragonBoard 845c board from a bricked software state. This instruction set is suggested to those who are experiences boot issues due to a corrution in the file system and/or other software components.
+This page outlines steps needed to recover your Qualcomm RB5 board from a bricked software state. This instruction set is suggested to those who are experiences boot issues due to a corrution in the file system and/or other software components.
```

Comment:
Typo
```suggestion
This page outlines steps needed to recover your Qualcomm RB5 board from a bricked software state. This instruction set is suggested to those who are experiencing boot issues due to a corruption in the file system and/or other software components.
```

Reply:
thanks for the review! I fixed them (and also in db845c which was initially copied here!)